sos/plugins/vdsm.py

```diff
@@ -130,7 +130,7 @@ def setup(self):
             )
 
         try:
-            res = self.call_ext_prog('vdsm-client Host getStorageDomains')
+            res = self.get_cmd_output_now('vdsm-client Host getStorageDomains')
```

Comment:
with `save=False`?

Reply:
Same here, this is useful data for support teams